app/main.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask
+from flask import request
+
+from src.cases_generator import CaseGenerator
+from src.full_solver import full_solver
+from src.solver import solve
+from utils.helpers import calculate_asl
+
+app = Flask(__name__)
+CASE_GEN = CaseGenerator(
+    "config/family_config.csv", filter="config/filter.yml", filter_bool=False
+)
+
+@app.route('/solve', methods=['POST'])
+def initial_solver():
+    case = request.json
```

Comment:
Sidi @omkhalil would like your opinion here. Right now just leaving the case as the request object. Unless you think we should add things to it such as `case` followed by the actual case within the object

Reply:
I think it's ok to leave it like this